### PR TITLE
backup: refactor some docs for easier understanding

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

According to some users feedback. the document of br is not easy to understand. this PR try to make it more clear.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

**Comment by allengaoo** [comment]
LGTM

File: br/external-storage.md [hunk 1]
```diff
@@ -210,9 +210,13 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
 
 ## BR 向 TiKV 发送凭证
 
+| 命令行参数 | 描述 |
+|:----------|:-------|
+| `--send-credentials-to-tikv` | 是否将 BR 获取到的权限凭证发送给 TiKV，默认是 true。|
```
Comment (shichun-0415) [inline br/external-storage.md:215]: external storage 这篇文档由[备份存储](https://docs.pingcap.com/zh/tidb/stable/backup-and-restore-storages)取代，请帮忙将这部分内容同步到备份存储，谢谢。

File: br/backup-and-restore-overview.md [hunk 2]
```diff
@@ -116,7 +119,7 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 | 全局临时表 | | 确保使用 BR v5.3.0 及以上版本进行备份和恢复，否则会导致全局临时表的表定义错误。 |
 | TiDB Lightning Physical Import| |上游数据库使用 TiDB Lightning Physical 方式导入的数据，无法作为数据日志备份下来。推荐在数据导入后执行一次全量备份，细节参考[上游数据库使用 TiDB Lightning Physical 方式导入数据的恢复](/faq/backup-and-restore-faq.md#上游数据库使用-tidb-lightning-physical-方式导入数据时为什么无法使用日志备份功能)。|
 
-#### 版本间兼容性
+### 版本间兼容性
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:122]: 下面的表格内容不太好维护，每次新发一个版本，都需要更新，而且，呈现方式不利于获取关键信息。
image

咱们能不能只写不兼容的情况：
- TiDB v6.1.0、v6.2.0 和 v6.3.0 快照备份可以恢复到 TiDB v6.0.0，但存在已知问题，即如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)。
- PITR 日志备份不可以恢复到 TiDB v6.2.0 及以下版本。

File: br/external-storage.md (hunk 1 above)
Comment (3pointer) [reply br/external-storage.md:215]: Done

File: br/backup-and-restore-overview.md [hunk 3]
```diff
@@ -15,6 +15,34 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
 - 审计业务的历史数据，满足司法审查的需求。
 - 复制 (Clone) 生产环境，方便问题诊断、性能调优验证、仿真测试等。
 
+## 使用须知
+
+本部分介绍使用 TiDB 备份恢复功能前的注意事项。包括使用限制和使用建议。
```
Comment (qiancai) [inline br/backup-and-restore-overview.md:20]:
```suggestion
本部分介绍使用 TiDB 备份恢复功能前的注意事项，包括使用限制和使用建议。
```

File: br/backup-and-restore-overview.md [hunk 4]
```diff
@@ -15,6 +15,34 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
...
+
+### 使用限制
+
+进行 PITR
+
```
Comment (qiancai) [inline br/backup-and-restore-overview.md:25]:
```suggestion
```

File: br/backup-and-restore-overview.md [hunk 5]
```diff
@@ -15,6 +15,34 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
...
+### 使用建议
+
+进行快照备份
+
+- 推荐在业务低峰时执行集群快照数据备份，这样能最大程度地减少对业务的影响。
+- 不推荐同时运行多个集群快照数据备份任务。不同的任务并行，不仅会导致备份的性能降低，影响在线业务，还会因为任务之间缺少协调机制造成任务失败，甚至对集群的状态产生影响。
+
+进行快照恢复
+
+- BR 恢复数据时会尽可能多地占用恢复集群的资源，因此推荐恢复数据到新集群或离线集群。应避免向正在提供服务的生产集群执行恢复，否则，恢复期间会对业务产生不可避免的影响。
```
Comment (qiancai) [inline br/backup-and-restore-overview.md:39]:
```suggestion
- BR 恢复数据时会尽可能多地占用恢复集群的资源，因此推荐恢复数据到新集群或离线集群。应避免恢复数据到正在提供服务的生产集群，否则，恢复期间会对业务产生不可避免的影响。
```

File: br/backup-and-restore-overview.md [hunk 6]
```diff
@@ -15,6 +15,34 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
...
+
+备份存储和网络配置
+
+- 推荐使用支持 S3、GCS 或 Azure Blob Storage 协议的存储系统保存备份数据；
```
Comment (qiancai) [inline br/backup-and-restore-overview.md:43]:
```suggestion
- 推荐使用支持 Amazon S3、GCS 或 Azure Blob Storage 协议的存储系统保存备份数据；
```

File: br/backup-and-restore-storages.md [hunk 7]
```diff
@@ -8,6 +8,26 @@ aliases: ['/docs-cn/dev/br/backup-and-restore-storages/','/zh/tidb/dev/backup-st
 
 TiDB 支持 Amazon S3、Google Cloud Storage (GCS)、Azure Blob Storage 和 NFS 作为备份恢复的存储。具体来说，可以在 `br` 的 `--storage` 或 `-s` 选项中指定备份存储的 URL。本文介绍不同外部存储服务中 [URL 的定义格式](#url-格式)、存储过程中的[鉴权方案](#鉴权)以及[存储服务端加密](#存储服务端加密)。
 
+## BR 向 TiKV 发送凭证
+
+| 命令行参数 | 描述 |
+|:----------|:-------|
+| `--send-credentials-to-tikv` | 是否将 BR 获取到的权限凭证发送给 TiKV，默认是 true。|
+
+在默认情况下，使用 Amazon S3、Google Cloud Storage (GCS)、Azure Blob Storage 存储时，BR 会将凭证发送到每个 TiKV 节点，以减少设置的复杂性。
```
Comment (shichun-0415) [inline br/backup-and-restore-storages.md:17]:
```suggestion
在默认情况下，使用 Amazon S3、Google Cloud Storage (GCS)、Azure Blob Storage 存储时，BR 会将凭证发送到每个 TiKV 节点，以减少设置的复杂性。该行为由参数 `--send-credentials-to-tikv` 控制，默认值为 `true`，即默认发送凭证到 TiKV 节点。
```
表格一般适用于呈现多个元素，这里只有一个参数，普通文字描述就可以？

Comment (3pointer) [reply br/backup-and-restore-storages.md:17]: 这里是希望所有信息统一使用表格，特意要求的。PTAL @benmaoer

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

File: br/backup-and-restore-overview.md [hunk 8]
```diff
@@ -15,6 +15,32 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
...
+## 使用须知
+
+本部分介绍使用 TiDB 备份恢复功能前的注意事项，包括使用限制和使用建议。
+
+### 使用限制
+
+- PITR 仅支持恢复到**全新的空集群**。
+- PITR 仅支持集群粒度的恢复，不支持对单个 database 或 table 的恢复。
+- PITR 不支持恢复用户表和权限表的数据。
+
```
Comment (benmaoer) [inline br/backup-and-restore-overview.md:27]:
```suggestion
PITR 恢复集群的任务运行期间，不支持打开日志备份任务，也不支持通过 TiCDC 同步数据到下游集群。
```

File: br/backup-and-restore-storages.md (hunk 7 above)
Comment (benmaoer) [reply br/backup-and-restore-storages.md]: LGTM
统一参数的格式，避免用户没有找到配置参数

File: br/backup-and-restore-overview.md (hunk 8 above)
Comment (shichun-0415) [reply br/backup-and-restore-overview.md:27]: I'll create another PR to add this sentece. Please remove the request change label. Thanks.

**Review by benmaoer — COMMENTED** [review]
LGTM

**Pull request merged**